docs/source/development-guide/docker.rst

```diff
@@ -0,0 +1,37 @@
+Docker Workflow
+----------------
+
+This page describes how to build and run a Docker Image Locally and in AWS.
+
+Building and Running a Docker Image Locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To build the image run the following command from the directory containing the Dockerfile. You might add -t option to tag your image
+and --rm to remove intermediate containers after the build is done.
+
+    `docker build -t <image name>:<tag name> --rm .`
```

Comment:
Do you want to give an example of how to build the image you added? You'll need the `-f Dockerfile.name` now because it isn't the default Dockerfile (sorry for the misdirection there)!

```suggestion
    `docker build -f examples/Dockerfile.efs -t <image name>:<tag name> --rm .`
```